### PR TITLE
ADEN-2211 Update interstitial test

### PR DESCRIPTION
**Pull request by drets**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestInterstitial.java [hunk 1]
```diff
@@ -23,10 +22,39 @@ public void interstitialAdScaled(
       String wikiName,
       String article,
       Dimension pageSize,
-      double scaledAdRatio
+      Dimension adSize,
+      boolean isAdShouldBeScaled
+  ) {
+    testInterstitial(wikiName, article, pageSize, adSize, isAdShouldBeScaled);
```
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestInterstitial.java]: What do you think about renaming `isAdShouldBeScaled` to `shouldAdBeScaled`?

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestInterstitial.java [hunk 2]
```diff
@@ -23,10 +22,39 @@ public void interstitialAdScaled(
...
+  }
+
+  @Test(
+      dataProviderClass = AdsDataProvider.class,
+      groups = "InterstitialMobile",
+      dataProvider = "interstitialMobile"
+  )
+  public void interstitialAdScaledMobile(
```
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestInterstitial.java]: Do we really need this wrapper-method? Can't we make `testInterstitial()` our test method which gets data from the data provider?

Comment (drets) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestInterstitial.java]: Yes. No. :imp:

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestInterstitial.java (hunk 1 above)
Comment (drets) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestInterstitial.java]: make sense :+1:

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestInterstitial.java (hunk 2 above)
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestInterstitial.java]: We discussed it more offline - if there is no other way let's live with it :crying_cat_face:

**Comment by nandy-andy** [comment]
LGTM :+1:

**Pull request merged**